orchard-provider-aws/src/main/scala/com/salesforce/mce/orchard/io/aws/activity/EmrActivity.scala

```diff
@@ -21,51 +24,55 @@ import com.salesforce.mce.orchard.system.util.InvalidJsonException
 case class EmrActivity(name: String, steps: Seq[EmrActivity.Step], clusterId: String)
     extends ActivityIO {
 
-  override def create(): Either[Throwable, JsValue] = Try {
-    val response = Client
-      .emr()
-      .addJobFlowSteps(
-        AddJobFlowStepsRequest
-          .builder()
-          .jobFlowId(clusterId)
-          .steps(
-            steps.map { step =>
-              StepConfig
-                .builder()
-                .name(name)
-                .actionOnFailure(ActionOnFailure.CONTINUE)
-                .hadoopJarStep(
-                  HadoopJarStepConfig
+  override def create(): Either[Throwable, JsValue] = catching(classOf[SdkException]) either
```

Comment:
I think we should return everything as Either, not just SdkException.

Reply:
Sure. Do you think RunTimeException would be sufficient?

Reply:
yeah, basically, what's wrong with `Try(...).toEither`?

Reply:
I agree Try(...).toEither is better than the catching idiom.  Latest commit refactor to this idiom.